packages/components/src/interfaces/native/Heading.ts

```diff
@@ -6,7 +6,9 @@ export type HeadingLevel = (typeof HeadingLevels)[number]
 
 export interface HeadingProps {
   type: "heading"
-  id: string // Used for anchor links
   content: TextProps[]
-  level: HeadingLevel
+  attrs: {
```

Comment:
adopted tiptap schema instead of bringing it up. this is also a **bugfix** because prosemirror assumes `attrs.<key>`. 

for example, `mergeCells` for table is done via `pm/table` and it uses `attrs.colspan`. hence, we can't destructively edit tiptap's schema. we can only add on